tutorials/cuda_mps/README.md

```diff
@@ -185,6 +205,83 @@ We vary the number of parallel inferences to show that MPS may not be beneficial
+```
+
+**3. Inside the container, be sure to set the following environment variables:**
```

Comment:
What container should be used? The example above has `holohub:v2.1`?

Could we provide a Dockerfile with these environment variables already set?

Reply:
@tbirdso Good point but this is anyway a temporary stop gap until we upgrade to CUDA 12.5+ officially. So, I think a dockerfile is not needed in the long-term.